test_olog.py

```diff
@@ -33,19 +36,102 @@
+                'state': 'Active'}
+    actual = user_cli.get_logbook(name=LOGBOOK_NAME)
+    assert actual == expected
 
 
 @vcr.use_cassette()
 def test_list_logs_by_logbook():
-    cli.list_logs(logbook=LOGBOOK_NAME)
+    expected = {'attachments': [],
+                'createdDate': 1487609015000,
+                'description': 'd\t\n\nCause:\nnull\n\nRepair:\nnull\n\nCorrective:\nnull',
+                'id': 21,
+                'level': 'Problem',
+                'logbooks': [{'id': 1,
+                              'logs': None,
+                              'name': 'Operations',
+                              'owner': None,
+                              'state': 'Active'}],
+                'modifiedDate': 1487609015000,
+                'owner': 'olog-user',
+                'properties': [{'attributes': {'Area': 'Global',
+                                               'Assign': 'Controls',
+                                               'BeamState': 'False',
+                                               'Contact': 'Kunal Shroff<shroffk@bnl.gov>',
+                                               'Device': 'M01',
+                                               'System': 'Diagnostics',
+                                               'TimeOccoured': '2017-02-20T16:43:23.408Z'},
+                                'groupingNum': 0,
+                                'id': 1,
+                                'logs': None,
+                                'name': 'fault'}],
+                'source': '130.199.219.79',
+                'state': 'Active',
+                'tags': [],
+                'version': '1'}
+    actual = user_cli.list_logs(logbook=LOGBOOK_NAME)
+    assert actual[0] == expected
 
 
 @vcr.use_cassette()
 def test_get_log():
-    cli.get_log(LOG_ID)
+    expected = {'attachments': [],
+                'createdDate': 1459083898000,
+                'description': 'Creating the first simple entry.\nInstructions for creating log entries\nusername: olog-user\npassword: 1234\n\n?????',
+                'id': 1,
+                'level': 'Info',
+                'logbooks': [{'id': 1,
+                              'logs': None,
+                              'name': 'Operations',
+                              'owner': None,
+                              'state': 'Active'}],
+                'modifiedDate': 1459083898000,
+                'owner': 'olog-user',
+                'properties': [],
+                'source': '127.0.0.1',
+                'state': 'Active',
+                'tags': [],
+                'version': '1'}
+    actual = user_cli.get_log(LOG_ID)
+    assert actual == expected
+
+
+@vcr.use_cassette()
+def test_create_logbook():
+    expected = {'id': 12,
+                'logs': None,
+                'name': 'Test',
+                'owner': 'test',
+                'state': 'Active'}
+    actual = admin_cli.put_logbook({'name': 'Test', 'owner': 'test'})
+    assert actual == expected
```

Comment:
Just in this case? Or for all other tests?

Reply:
Otherwise if someone runs the tests twice, they will get failures the second time. Or do you think people should deploy a _clean_ instance of the VM every time, so that the ID counter starts at some standard initial value?

Reply:
I agree we should not rely on the id assignment, and it's hard to reproduce the clean machine. I cannot reach the link to the VM provided by @shroffk before. See my updated test in 
cc40f77.